src/errors/types.js

```diff
@@ -146,5 +146,12 @@ export const RUNTIME_ERRORS = {
     uncaughtErrorInReporter:                            'E1058',
     roleInitializedWithRelativeUrl:                     'E1059',
     typeScriptCompilerLoadingError:                     'E1060',
-    cannotCustomizeSpecifiedCompilers:                  'E1061'
+    cannotCustomizeSpecifiedCompilers:                  'E1061',
+    browserConnectionError:                             'E1062'
+};
+
+export const CONNECTION_ERROR_HINTS = {
+    tooHighConcurrencyFactor: 'H1',
```

Comment:
Error codes were added to describe them in future (similar as - [DevExtreme](https://js.devexpress.com/Documentation/ApiReference/UI_Widgets/Errors_and_Warnings/)).

Hints is internal thing. So, I propose to convert it to enum.